Orion/Npcs/INpcService.cs

```diff
@@ -37,7 +37,21 @@ public interface INpcService : IService
 		event EventHandler<NpcKilledEventArgs> NpcKilled;
 
 		/// <summary>
+<<<<<<< cc33da6b11c8d846a6d3e46ac7c60f9e027bfc6e

```

Comment:
You got merge crap in your NPC service definition, this won't compile.  Fix this and the build should work (Y)